region/maridia/inner-yellow.json

```diff
@@ -2205,101 +2199,175 @@
+                     ]}
+                    
                     ]}
                   ],
-                  "note": "Pseudo screw or shinespark the left pirate, then IBJ out",
-                  "devNote": "If you can kill them without being hurt, you can take path 2->3->1."
+                  "note": "Pseudo screw the left pirate, or clear the left side with shinesparks, then IBJ out",
+                  "devNote": [
+                    "If you can kill them without being hurt, you can take path 2->3->1.",
+                    "Shinecharging pirates is easier to setup with Morph."
+                  ]
                 },
                 {
                   "name": "Jump into IBJ",
                   "notable": false,
-                  "requires": ["canJumpIntoIBJ"]
-                },
-                {
-                  "name": "Shinespark",
-                  "notable": false,
-                  "requires": [
-                    {"canShineCharge": {
-                      "usedTiles": 19,
-                      "shinesparkFrames": 30,
-                      "openEnd": 0
-                    }}
-                  ]
+                  "requires": ["canJumpIntoIBJ"],
+                  "failures": [
+                    {
+                      "name": "Fail Bomb Jump",
+                      "cost": [
+                        { "enemyDamage": { "enemy": "Pink Space Pirate (standing)", "hits": 1, "type": "laser" } }
+                      ]
+                    }
+                  ],
+                  "note": "Let the wall pirate jump to the otherside of the room.  Shooting the standing pirate (with anything) will prevent it from attacking."
                 },
                 {
-                  "name": "Kill one then Shinespark",
+                  "name": "Shinespark Clear With Pirate Hit",
                   "notable": false,
                   "requires": [
-                    {"canShineCharge": {
-                      "usedTiles": 21,
-                      "shinesparkFrames": 30,
-                      "openEnd": 0
-                    }},
-                    "Charge",
+                    "canCarefulJump",
                     {"enemyDamage": {
                       "enemy": "Pink Space Pirate (standing)",
-                      "type": "contact",
+                      "type": "laser",
                       "hits": 1
                     }}
                   ],
-                  "note": "Pseudo screw the bottom pirate to make a larger runway, then shinespark out",
-                  "devNote": "If you can kill them without being hurt, you can take path 2->3->1."
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": [
+                        {"canShineCharge": {
+                          "usedTiles": 21,
+                          "shinesparkFrames": 33,
+                          "openEnd": 0
+                        }}
```

Comment:
I'm thinking the enemy damage should be inside the obstacle. Should there be a 19 tile runway `or` the laser damage?

Reply:
Is there a reason pseudo screw was removed?

Reply:
Its logically irrelevant.  With or without pseudo, you take one pirate hit then shortcharge.  It would remove the failure listing.  There could be an `avoidableWith` property to failures.

The only real point of separation between 21tile and 19tile shinespark clear is the note.  They could be combined.